index.js

```diff
@@ -108,6 +109,10 @@ async function send (ctx, path, opts = {}) {
     // and not require a trailing slash for directories,
     // so that you can do both `/directory` and `/directory/`
     if (stats.isDirectory()) {
+      if (directorySlash && !trailingSlash) {
+        ctx.redirect(ctx.request.path + '/')
```

Comment:
LGTM 👍, can you write the test !! 

```suggestion
        ctx.redirect(`${ctx.request.path}/`)
```